cypress/integration/psiDetailView.spec.js

```diff
@@ -4,7 +4,7 @@ describe('Tests the PSI View', () => {
     cy.kcLogin('test-moh');
     cy.visit('/psi-view');
     cy.contains('Add Post Secondary Institution').click({ force: true });
-    cy.wait(1000);
+    cy.wait(2000);
```

Comment:
Do we need to increase the timeout? Tests are failing either way

Reply:
Cypress test is failing and there is no code change related to this section.